backend/utils.py

```
@@ -0,0 +1,343 @@
+        )
+        
+        # Return structured response similar to Google Vision format
+        return {
+            "responses": [{
+                "labelAnnotations": [{"description": "DIY item"}],
+                "textAnnotations": [{"description": chat_response.choices[0].message.content}],
+                "mistral_analysis": chat_response.choices[0].message.content
+            }]
+        }
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)